sdk/core/azure-core/src/main/java/com/azure/core/exception/HttpResponseInnerError.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.exception;
+
+
+/**
+ * The inner error of a {@link HttpResponseError}.
+ */
+public final class HttpResponseInnerError {
+
+    private String code;
+    private HttpResponseInnerError innerError;
```

Comment:
Are none of the properties required in this class?

Reply:
They're all optional as defined in the [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/Guidelines.md#errorresponse--object).